### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by LargePrime** [gmariani/cv-minerva#67]
opening any sol gives

Error

Error reading file:
SecurityError: Failed to load worker script at "http://apps.coursevector.com/minerva/js/parsers/SOLReaderWorker.js"
(:0)

**Comment by PrincessRTFM** [comment]
Specifically:

```
Uncaught DOMException: Failed to construct 'Worker': Script at 'http://apps.coursevector.com/minerva/js/parsers/SOLReaderWorker.js' cannot be accessed from origin 'https://mariani.life'.
    at SOL.deserialize (http://cdn.coursevector.com/minerva/4.1.3/js/main.min.js:236:392)
    at FileReader.z.onload (http://cdn.coursevector.com/minerva/4.1.3/js/main.min.js:309:185)
```

Looks like it's forbidding a cross-domain request. cURL shows a `301 Moved Permanently` to `https://mariani.life/projects/minerva/js/parsers/SOLReaderWorker.js` instead, so I'm guessing you have a domain forward. Just grep your source for `coursevector.com` and replace the URLs with the new domain.

**Comment by Ilani76** [comment]
No offense to you, but what?

I'm having the same problem, but I'm new to programming and using code in general. So I'm 100% sure what to do to fix the error.

**Comment by Ilani76** [comment]
No offense to you, but what?

I'm having the same problem, but I'm new to programming and using code in general. So I'm 100% sure what to do to fix the error.

**Comment by Ilani76** [comment]
No offense to you, but what?

I'm having the same problem, but I'm new to programming and using code in general. So I'm not 100% sure what to do to fix the error.

*Meant to say, I'm not 100% sure.

**Comment by PrincessRTFM** [comment]
JavaScript is generally forbidden to make requests to a site with a different domain than the one the web page is from. In this case, `apps.coursevector.com` is the request domain, and `mariani.life` is the web page's domain. They're different, so the request is blocked by the browser. Thing is, the requested script that's been blocked (`http://apps.coursevector.com/minerva/js/parsers/SOLReaderWorker.js`) is also available at `http://mariani.life/minerva/js/parsers/SOLReaderWorker.js`. The domain there is the same, so that would be allowed.

tl;dr: The domain changed recently, which broke scripts requesting resources from the old domain, since the code hasn't been updated for the new domain. The fix is simple: find everywhere that a resource is being requested from `*.coursevector.com` and change the URL to use `mariani.life` instead.

Mind, this depends on the resources all being _available_ at `mariana.life` too.

**Comment by gmariani** [comment]
Sorrry, have been busy lately. I've changed it over to HTTPS only and updated the paths to mariani.life. Once cloudfront clears it's caches the rest of the issues should go away. If you clear your cache it should be back up. Sorry for the inconvenience.
